ports/raspberrypi/common-hal/socketpool/Socket.c

```diff
@@ -350,11 +356,11 @@ STATIC mp_uint_t lwip_raw_udp_send(socketpool_socket_obj_t *socket, const byte *
         len = 0xffff;
     }
 
-    MICROPY_PY_LWIP_ENTER
+    MICROPY_PY_LWIP_ENTER;
```

Comment:
(I inquired why micropython wrote the macro so that it was used without a trailing semicolon but there didn't seem to be a definitive answer)

Reply:
Yeah, sorry. I admit I was being a tad precious by changing the semicolon just because it annoyed me.